mlflow/server/__init__.py

```diff
@@ -13,6 +13,13 @@
     get_model_version_artifact_handler,
 )
 from mlflow.utils.process import exec_cmd
+from ih_telemetry.init import initialize_telemetry
+from ih_telemetry.middleware import make_app_with_telemetry
+
+
+def post_fork(server, worker):
```

Comment:
i think in order for gunicorn to pick this up by default it would need to be in a file called `gunicorn.conf.py`, see these docs: https://docs.gunicorn.org/en/latest/settings.html#

Reply:
this wasn't addressed, and i still think this function won't actually be executed, unless you can see from the UAT logs that it is

Reply:
Tried to address this. Sorry for missing it earlier. Changes were deployed to UAT - https://ci.us-east-1.uat.grnds.com/#/namespaces/deploy/pipelineruns/mlflow-pr-9a32c11a?pipelineTask=slack-notify-start&step=slack-notify-start